To: vim-dev@vim.org
Subject: Patch 7.0.045 (extra)
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=ISO-8859-1
Content-Transfer-Encoding: 8bit
------------

Patch 7.0.045 (extra)
Problem:    Win32: Warnings when compiling OLE version with MSVC 2005.
Solution:   Move including vim.h to before windows.h. (Ilya Bobir)
Files:	    src/if_ole.cpp


*** ../vim-7.0.044/src/if_ole.cpp	Wed May  3 23:18:50 2006
--- src/if_ole.cpp	Sat Jun 24 13:36:52 2006
***************
*** 13,23 ****
   * See os_mswin.c for the client side.
   */
  
  #include <windows.h>
  #include <oleauto.h>
  
  extern "C" {
- #include "vim.h"
  extern HWND s_hwnd;
  extern HWND vim_parent_hwnd;
  }
--- 13,26 ----
   * See os_mswin.c for the client side.
   */
  
+ extern "C" {
+ #include "vim.h"
+ }
+ 
  #include <windows.h>
  #include <oleauto.h>
  
  extern "C" {
  extern HWND s_hwnd;
  extern HWND vim_parent_hwnd;
  }
*** ../vim-7.0.044/src/version.c	Tue Aug  8 16:47:38 2006
--- src/version.c	Tue Aug  8 17:04:14 2006
***************
*** 668,669 ****
--- 668,671 ----
  {   /* Add new patch number below this line */
+ /**/
+     45,
  /**/

-- 
From "know your smileys":
 %	Bike accident.  A bit far-fetched, I suppose; although...
             o      _     _         _
     _o     /\_   _ \\o  (_)\__/o  (_)
   _< \_   _>(_) (_)/<_    \_| \   _|/' \/
  (_)>(_) (_)        (_)   (_)    (_)'  _\o_

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\        download, build and distribute -- http://www.A-A-P.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
